subprojects/docs/src/main/resources/header.html

```diff
@@ -88,81 +88,66 @@
             <input type="search" name="q" id="search-input" class="search-input" placeholder="Search the docs"/>
         </div>
         <ul>
-            <li class="docs-home-link"><a href="/userguide/userguide.html">Gradle Overview</a> <div class="pdf-link-container">
+            <li class="docs-home-link"><a href="/userguide/userguide.html">Documentation Home</a> <div class="pdf-link-container">
                 <a href="/userguide/userguide.pdf">
                     <svg width="10px" height="12px" viewBox="0 0 10 12" version="1.1" xmlns="http://www.w3.org/2000/svg"><path d="M6.01302083,0.857142857 C6.2421875,0.857142857 6.4296875,1.05 6.4296875,1.28571429 L6.4296875,5.14285714 L8.875,5.14285714 C9.05989583,5.14285714 9.15364583,5.37321429 9.0234375,5.50714286 L5.29427083,9.35357143 C5.13020833,9.52232143 4.8671875,9.52232143 4.703125,9.35357143 L0.979166667,5.50714286 C0.848958333,5.37321429 0.940104167,5.14285714 1.12760417,5.14285714 L3.57291667,5.14285714 L3.57291667,1.28571429 C3.57291667,1.05 3.76041667,0.857142857 3.98958333,0.857142857 L6.01302083,0.857142857 Z M6.01302083,0 L3.98958333,0 C3.29947917,0 2.73958333,0.575892857 2.73958333,1.28571429 L2.73958333,4.28571429 L1.12760417,4.28571429 C0.203125,4.28571429 -0.265625,5.4375 0.390625,6.1125 L4.11458333,9.95892857 C4.60416667,10.4625 5.39583333,10.4625 5.88541667,9.95892857 L9.61197917,6.11517857 C10.265625,5.44285714 9.80208333,4.28839286 8.875,4.28839286 L7.26302083,4.28839286 L7.26302083,1.28571429 C7.26302083,0.575892857 6.70052083,0 6.01302083,0 Z M10,11.6785714 L10,11.4642857 C10,11.2875 9.859375,11.1428571 9.6875,11.1428571 L0.3125,11.1428571 C0.140625,11.1428571 0,11.2875 0,11.4642857 L0,11.6785714 C0,11.8553571 0.140625,12 0.3125,12 L9.6875,12 C9.859375,12 10,11.8553571 10,11.6785714 Z"></path></svg>
                     PDF
                 </a>
             </div></li>
-            <li><a href="/userguide/getting_started.html">Getting Started</a></li>
+            <li><a href="https://guides.gradle.org">Tutorials</a></li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#gradle-api" aria-expanded="false" aria-controls="gradle-api">Gradle API</a>
+                <ul id="gradle-api">
+                    <li><a href="/javadoc/">Javadoc</a></li>
+                    <li><a href="/dsl/">Groovy DSL Reference</a></li>
+                    <li><a href="https://gradle.github.io/kotlin-dsl-docs/api/" target="_blank">Kotlin DSL Reference</a></li>
+                    <li><a href="/userguide/groovy_build_script_primer.html">Groovy DSL Primer</a></li>
```

Comment:
OK, so this Kotlin DSL Primer is causing understandable problems. The main issue is that the content itself doesn't fit cleanly into explanation style or reference style. On top of that, it bears little relation to the Groovy DSL Primer. 

I think we absolutely need to give it a new name. I just haven't invested any effort in doing so yet. I was considering calling it the "Kotlin DSL Reference", but that would just be confusing because it's nothing like the Groovy DSL Reference.

Ideally, the new name wouldn't pigeonhole the document into a particular part of the nav, and certainly not under Gradle API. The reason I considered putting the Groovy DSL Primer under Gradle API was because it explicitly explains how the DSL maps to the API docs. That's not what the Kotlin DSL Primer does.